### PR TITLE
Zone local physics

### PR DESCRIPTION
**Pull request by sethalves**
- zones with children run their own local entity simulation

### CODE REVIEW

**Comment by sethalves** [comment]
This isn't done --  I've opened this PR so that andrew can tell me where I've gone wrong.

**Comment by hifi-gustavo** [comment]
Build console output for commit: b03220e2a8381ec0fd215df3f1e0ed49592b08ed available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b39e278689e41ce9e9795e4d18d4fa64cd231782 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d8939659efeeecb58f536a39694a677fb60a8c6b available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: db1f09600eae88eff5d180e870a86260b68f35ef available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: db1f09600eae88eff5d180e870a86260b68f35ef available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a96924d0fef88d8b85eefe180fe62acb969fc024 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4d2aa4e6bdb57977a1a9490ef23ef91ff2d1a867 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3661f04ce9b510899a81557667fa46239d498da1 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9652a41c75f998787234b4053a3e8a2552a58c8b available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5751f77dc51c8ba84fc91defa9c7250808b7e8c6 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fe3c52e23f21490266714c6f42464e42d45602ae available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5eefb74a46eb0f52cd25a6773698f627181336db available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b9ff27e2140759c6f246a4ce8c87e97069285e31 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 13a199b3c25e5bb48ec3888b9bebaf3134baf4b6 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 23a5a677e094e49f041c225b2248866291988e15 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8b116cdeae9d0757a8bcefedb3d7b37898bf3dac available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2a0fecad7f5e175b33563ddca8f3cb7bc92954c5 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-2a0feca.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-2a0feca.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: be0b213aa3fff673ad3f24eb3fc682ba89479c6d available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-be0b213.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-be0b213.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: be0b213aa3fff673ad3f24eb3fc682ba89479c6d available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-be0b213.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-be0b213.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a7746e4266e9f0f59124b7d7fcb49cc39b364332 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-a7746e4.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-a7746e4.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ef4d25a511419e0cbbb0a26f6e1056a2446f306d available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-ef4d25a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-ef4d25a.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0833bbfecf19d00bbada88d976f1ee069e53eb3f available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-0833bbf.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-0833bbf.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 55d7c41fb77d094301c9997ff74db4475ae3d5fa available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-55d7c41.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-55d7c41.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b5f668fd3e71ce80e67f6d0637d6cf0782be99db available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-b5f668f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-b5f668f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0570a072e55529c8876ffc709526a6b6b7896ac3 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-0570a07.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-0570a07.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc83746380636c91bd357989066d4f96b386541c available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc83746380636c91bd357989066d4f96b386541c available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc83746380636c91bd357989066d4f96b386541c available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc83746380636c91bd357989066d4f96b386541c available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-fc83746.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: cf8563402275886a14e01259695db4c2eb6eb55a available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-cf85634.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-cf85634.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8206761a987d338e517998094309fbb97c1382e6 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-8206761.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-8206761.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 896f5161726b2c303e268db64d257693954faaf8 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-896f516.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-896f516.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0b5975d4a9b47f2b6709e0d0b7cdc3b94c721191 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-0b5975d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-0b5975d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2e2c3f3fd6d1660fb0a203eb0905d2acae971676 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-2e2c3f3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-2e2c3f3.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6fc513294e5cc133132c939e880a705d77c49304 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-6fc5132.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-6fc5132.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1306819aa025dadbd00825ca033359f517c51aba available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-1306819.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-1306819.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: de936dafd1364ff2a53610b65948c1098c23bf08 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-de936da.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-de936da.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b1834276630d741b7cdf99c6026e5cfafa838334 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-b183427.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-b183427.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 781b4992fd773b76a75b8bd3abb39e3dd1ca8746 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-781b499.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-781b499.dmg

File: scripts/system/html/entityProperties.html [hunk 1]
```diff
@@ -541,6 +541,42 @@
                     <div><input type="number" class="blue" id="property-light-color-blue"><label for="property-light-color-blue">Blue:</label></div>
                 </div>
             </div>
+        </div>
+        <div class="zone-group zone-section skybox-section property url ">
+            <label for="property-zone-skybox-url">Skybox URL</label>
```
Comment (thoys) [inline scripts/system/html/entityProperties.html]: Wasn't this property already there? I think something gone wrong merging

**Comment by hifi-gustavo** [comment]
Build console output for commit: 94974edf7c50526aa3b8812a97957e58b6cc74f7 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-94974ed.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-94974ed.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 69763ebccee65f72a5326f45bdfd14f0da64eec1 available at: https://gist.github.com/256470339b16da5bbbaf895702f772de<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-69763eb.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7905-69763eb.dmg

**Comment by ZappoMan** [comment]
:(

**Comment by sethalves** [comment]
I'll keep the branch!  Just trying to unclutter PRs.

**Pull request closed without merging**